sovrin_client/agent/agent_verifier.py

```diff
@@ -53,3 +59,21 @@ def __init__(self, verifier: Verifier):
                              'claim issued by: {}'.format(", ".join(
                 sorted([sk.issuerId for sk in proof.schemaKeys]))))
             await self._postClaimVerif(claimName, link, frm)
+
+    def sendProofReq(self, link: Link, proofReqSchemaKey):
+        if self._proofRequestsSchema and (
+                    proofReqSchemaKey in self._proofRequestsSchema):
+            proofRequest = self._proofRequestsSchema[proofReqSchemaKey]
+            op = OrderedDict([
```

Comment:
It was used in same manner on other functions such as `sendProofAsync`. If it can take care of same format while receiving the message as well, then it's fine, I can change it to normal dictionary.